common/adapter/error.go

```diff
@@ -81,3 +83,52 @@ func ToProtoError(in error) error {
 
 	return st.Err()
 }
+
+// ToProtoError converts gRPC error to Thrift error.
+func ToThriftError(st *status.Status) error {
```

Comment:
I guess this would go away after grpc migration is done?

Reply:
Yes, entire `adapter` will go away.